credentials/xds/xds_client_test.go

```diff
@@ -46,8 +47,11 @@ import (
 const (
 	defaultTestTimeout      = 1 * time.Second
 	defaultTestShortTimeout = 10 * time.Millisecond
-	defaultTestCertSAN      = "abc.test.example.com"
-	authority               = "authority"
+	// defaultTestTimeout       = 1 * time.Hour
+	// defaultTestShortTimeout  = 10 * time.Hour
+	defaultTestCertSAN       = "abc.test.example.com"
+	defaultTestCertSANSPIFFE = "spiffe://foo.bar.com/9eebccd2-12bf-40a6-b262-65fe0487d453"
+	authority                = "authority"
```

Comment:
nit: Should we alphabetize the constants?

Reply:
Done